docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM metabase/metabase:v0.44.6
+RUN apk update && apk add curl jq
+COPY scripts /app/scripts
+RUN chmod +x /app/scripts/metabase/*.sh
+RUN chmod +x /app/scripts/user/*.sh
+RUN chmod +x /app/scripts/database/*.sh
+RUN chmod +x /app/scripts/reports/*.sh
```

Comment:
Instead of adding 4 times chmod, you can add a single line command:

find /app/scripts -type f -iname "*.sh" -exec chmod +x {} \;
OR
chmod +x /app/scripts/**/*.sh

Reply:
chmod +x /app/scripts/**/*.sh

Reply:
With this script, we are trying to establish a sample report which will be available for users to see the capability of metabase visualisations. 
We selected two sets of reports that will be executed during the metabase startup and will be available just as a reference. 


As the purpose here is to create a container for metabase, I believe, if we try to create and append an automatic reporting capabilities here, it might end up getting more complicated. 

If we need a solution to automate report generations through configuration and not just using metabase ui, I suggest we take it as a separate initiative. 

We can discuss further on this.

Reply:
Here is an inspiration on how to do this: https://blog.koverhoop.com/import-and-export-contents-of-one-metabase-instance-to-another-b79d1858ca37

I really don't like the idea of using CURL to insert SQL. At minimum this has to be wrapped inside a piece of code that triggers the post. I don't see this requiring more than 1 day to write.

Reply:
You can also explore using curl command with "file" input. See: https://stackoverflow.com/questions/6408904/send-request-to-curl-with-post-data-sourced-from-a-file